docs/ru/sql-reference/statements/create/row-policy.md

```diff
@@ -1,6 +1,6 @@
 ---
-toc_priority: 7
-toc_title: Политика доступа
+toc_priority: 8
+toc_title: "\u041f\u043e\u043b\u0438\u0442\u0438\u043a\u0430\u0020\u0434\u043e\u0441\u0442\u0443\u043f\u0430"
```

Comment:
Why?

Reply:
Sorry, I wanted to move the Overview section at the beginning of section. It looks like I was overworked and made a mistake.